scripts/migrations/migrate_2.3.0.js

```diff
@@ -42,8 +42,15 @@ const config = {
   remove: [],
   skipSelectors: {},
   facetsToInit: {
-    ExchangeHandlerFacet: { init: [], constructorArgs: [EXCHANGE_ID_2_2_0[network]] },
-  }, // must match nextExchangeId at the time of the upgrade
+    ExchangeHandlerFacet: { init: [], constructorArgs: [EXCHANGE_ID_2_2_0[network]] }, // must match nextExchangeId at the time of the upgrade
```

Comment:
Any way to extract EXCHANGE_ID_2_2_0 for the current network at the time of the update?